modules/developer_manual/examples/core/scripts/php/dav/public_files/view_public_link.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use GuzzleHttp\Client;
+
+require_once('vendor/autoload.php');
+
+// Configure the basic client
+$client = new Client([
+    'base_uri' => '{oc-examples-server-url}/remote.php/dav/',
+]);
+
+$share_token = '<share_token>';
+
+try {
+    $response = $client->request('PROPFIND', "public-files/${share_token}", [
+        'auth' => [
+            '{oc-examples-username}',
```

Comment:
same as above

Reply:
Thanks for the clarification on both points.